### ISSUE TITLE
Not posible to disable `unknown-option-value`

### ISSUE THREAD

**Issue by moylop260** [pylint-dev/pylint#10188]
### Bug description

Create a python file `my_pylint_test.py` with the following content:
```python
# pylint: disable=class-camelcase
```

### Configuration

```ini
# .pylintrc file
[MESSAGES CONTROL]
disable=all
enable=consider-using-dict-items
```

### Command used

```shell
pylint my_pylint_test.py
```

### Pylint output

```txt
************* Module my_pylint_test
my_pylint_test:1:0: W0012: Unknown option value for 'disable', expected a valid pylint message and got 'class-camelcase' (unknown-option-value)
```

### Expected behavior

Since that `disable` option is configured with`all`

It should omit other checks different from already enabled

### Pylint version

```shell
pylint 3.3.3
astroid 3.3.8
Python 3.13.1 (main, Dec  3 2024, 17:59:52) [Clang 16.0.0 (clang-1600.0.26.4)]
```

### OS / Environment

`MacOSX Sequoia 15.2 (24C101)`

### Additional dependencies

```requirements.txt
astroid==3.3.8
dill==0.3.9
isort==5.13.2
mccabe==0.7.0
platformdirs==4.3.6
pylint==3.3.3
tomlkit==0.13.2
```

**Comment by jacobtylerwalls** [comment]
Thanks for the issue, but this behavior is intentional, see #9403.
